enrolments/tests/utils/test_load_initial_data.py

```diff
@@ -12,20 +12,17 @@ def setUp(self):
         Student.objects.bulk_create(
             [
                 Student(first_name="Marlin", last_name="Fish", role=Student.PARENT),
-                Student(first_name="Nemo", last_name="Fish", role=Student.CHILD),
+                Student(
+                    first_name="Nemo",
+                    last_name="Fish",
+                    role=Student.CHILD,
+                    date_of_birth="2002-09-28",
```

Comment:
these objects are created in the test setup just to ensure they're deleted by the script, so we don't need to update this!